# goal 
uint8 EndEffectorGoal = 0
uint8 JointGoal = 1

geometry_msgs/Pose goal_pose
sensor_msgs/JointState goal_joint_state
uint8 type

octomap_msgs/Octomap octomap

#if has_attached_object set, attached_object should contain valid data for the attached object
bool has_attached_object
moveit_msgs/AttachedCollisionObject attached_object

bool execute_path
---
# result
bool success
trajectory_msgs/JointTrajectory trajectory
---
# feedback
float64 elapsed_time
